static/css/github-create-branch.css

```diff
@@ -51,8 +51,14 @@
   right: 4px;
 }
 .select2-loader {
+  display: flex;
```

Comment:
The spinner looks a bit cramped, can we add some padding for a bit of vertical space?
You might need to add extra room for the results too so it doesn't bounce around in size..... maybe throw some screenshot options at jajajade?

Reply:
Padding has been added.
And as for bouncing effect, that will happen in most cases because we don't know the number of repos that will be returned. So, lets say if we make the spinner about the same height as the original dropdown, and now if the search result is just 1 repo, it will still do the jumpy effect cause the dropdown's height will decrease after finishing loading.